eng/Sharpliner.CI/ProjectBuildSteps.cs

```diff
@@ -12,7 +12,7 @@ class ProjectBuildSteps(string project) : StepLibrary
     [
         StepTemplate(Pipelines.TemplateLocation + "install-dotnet-sdk.yml", new()
         {
-            { "version", "8.0.204" }
+            { "version", "8.0.403" }
```

Comment:
@premun would be nice to have a utility to expose a `GlobalJson` type that we could leverage for pipelines (this type would be static and read value for the local `globa.json` file

Reply:
Yeah, that sonuds like a good idea. I wonder if we should make that a feature of the library or just an internal feature for this repo